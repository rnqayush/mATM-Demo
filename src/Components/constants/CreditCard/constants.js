import ScreenConstants from '../screens-view-en.json';
import { translate } from '../../l10n';

const constants = ScreenConstants.HAVE.CARDS;
const apiConstants = ScreenConstants.CREDIT_CARD;

export const CARD_STATUS = {
  ACTIVE: 'NORM',
  TEMPORARILY_BLOCKED: 'FSFT',
  PERMANENTLY_BLOCKED: 'FHRD',
};
export const CARD_STGEN = {
  ACTIVE: 'NEW',
  TEMPORARILY_BLOCKED: 'NEW',
  PERMANENTLY_BLOCKED: 'FHRD',
};

export const RUPEE_SYMBOL = '\u20B9';

export const CARD_ACTION = {
  INCREASE_CREDIT_LIMIT: { label: constants.INCREASE_CREDIT_LIMIT, httpStatus: 'INCREASE_CREDIT_LIMIT' },
  PAY_CREDIT_BILL: { label: constants.PAY_CREDIT_BILL, httpStatus: 'PAY_CREDIT_BILL' },
  BLOCK: { label: translate('CREDIT_CARD.CC_TEMPORARY'), httpStatus: 'BLOCKED' },
  UNBLOCK: { label: constants.UNBLOCK, httpStatus: 'UNBLOCKED' },
  CANCEL: { label: constants.CANCEL },
  PERMANENTLY_BLOCKED: { label: translate('CREDIT_CARD.CC_PERMANENTBLOCK'), httpStatus: 'PERMANENTLY_BLOCKED' },
  REPLACE_CARD: { label: constants.REPLACE_CARD, httpStatus: 'REPLACE_CARD' },
  CHANGE_PIN: { label: constants.CHANGE_PIN },
  SET_PIN: { label: translate('CREDIT_CARD.CC_SETPIN'), httpStatus: 'SET_PIN' },
  UNMASK_CREDIT_CARD_NUMBER: { label: constants.UNMASK_CREDIT_CARD_NUMBER, httpStatus: 'UNMASK_CREDIT_CARD_NUMBER' },
  GENERATE_PIN: { label: constants.GENERATE_PIN },
  GENERATE_PHYSICAL_CARD: { label: constants.GENERATE_PHYSICAL_CARD },
  TEMP_BLOCKED: { label: constants.TEMP_BLOCKED, httpStatus: 'TEMP_BLOCKED' },
  // SHOW_HISTORY: { label: constants.SHOW_HISTORY, httpStatus: 'SHOW_HISTORY' },
  PAY_OUTSTANDING: { label: constants.PAY_OUTSTANDING, httpStatus: 'PAY_OUTSTANDING' },
  CONVERT_TO_EMI: { label: constants.CONVERT_TO_EMI, httpStatus: 'CONVERT_TO_EMI' },
  FOR_CLOSE_EMI_CANCEL: { label: constants.FOR_CLOSE_EMI_CANCEL, httpStatus: 'FOR_CLOSE_EMI_CANCEL' },
  CREDIT_CARD_EMI: { label: constants.CREDIT_CARD_EMI, httpStatus: 'CREDIT_CARD_EMI' },
  DOWNLOAD_E_STATEMENT: { label: constants.DOWNLOAD_E_STATEMENT, httpStatus: 'DOWNLOAD_E_STATEMENT' },
  CREDIT_CARD_APPLY: { label: constants.CREDIT_CARD_APPLY, httpStatus: 'CREDIT_CARD_APPLY' },
  POSHVINE: { label: constants.POSHVINE, httpStatus: 'POSHVINE' },
  CREDIT_CARD_TRANSACTION: { label: constants.CREDIT_CARD_TRANSACTION, httpStatus: 'CREDIT_CARD_TRANSACTION' },
  IMAGE_CARD: { label: translate('CREDIT_CARD.CC_IMAGECARD_CTA_TITLE'), httpStatus: 'CC_IMAGECARD_CTA_TITLE' },
  BALANCE_TRANSFER: { label: translate('CREDIT_CARD.CC_BT_CTA_TITLE'), httpStatus: 'CC_BT_CTA_TITLE' },
  APPLY_ADDON: { label: translate('CREDIT_CARD.CC_APPLY_ADDON'), httpStatus: 'CC_APPLY_ADDON_CTA_TITLE' },
};

export const BLOCK_CARD_TITLE = constants.BLOCK_PERMANENTLY;
export const REISSUE_CARD_TITLE = constants.REISSUE_CARD;
export const SYSTEM_ID = apiConstants.OPTIMUS;
export const ORIGIN = apiConstants.EXTERNAL;
export const DATA_VERSION = 0;
export const REQUEST_TYPE = apiConstants.CARD_STATUS;
export const REFERENCE_VALUE_C = apiConstants.REFERENCE_C;
export const REFERENCE_VALUE_U = apiConstants.REFERENCE_U;

export const BLOCK_REASONS = [
  { key: 'LOST_OR_STOLEN', value: constants.LOST_OR_STOLEN },
  { key: 'DAMAGED', value: constants.DAMAGED },
];

export const REQUEST_HEADER = {
  Header: {
    MessageID: '9d634eff-4695-46e0-95be-1eedee5afa42',
    SystemID: SYSTEM_ID,
    Origin: ORIGIN,
  },
};
export const PROVIDERS = {
  VISA: constants.VISA,
  RUPAY: constants.RUPAY,
  MASTER: constants.MASTER,
};

export const CARD_VARIANT = {
  PLATINUM: 'Platinum',
  SIGNATURE: 'Signature',
  BUSINESS: 'SignatureBusiness',
  CLASSIC: 'Classic',
};

export const WARNINGS = {
  BLOCK_REASON: constants.BLOCK_REASON,
  REISSUE_REASON: constants.REISSUE_REASON,
};

export const LESS_OPACITY_TEXT_CARDS = [
  CARD_STATUS.EXPIRED,
  CARD_STATUS.TEMPORARILY_BLOCKED,
  CARD_STATUS.PERMANENTLY_BLOCKED,
  CARD_STATUS.CLOSED,
];

export const LOCKED_CARD = [
  CARD_STATUS.EXPIRED,
  CARD_STATUS.TEMPORARILY_BLOCKED,
  CARD_STATUS.PERMANENTLY_BLOCKED,
  CARD_STATUS.CLOSED,
  CARD_STATUS.INACTIVE,
];

export const DISABLED_CARDS = [
  CARD_STATUS.CLOSED,
  CARD_STATUS.EXPIRED,
  CARD_STATUS.PERMANENTLY_BLOCKED,
  CARD_STATUS.INACTIVE,
];

export const TOGGLE = {
  ON: constants.ON,
  OFF: constants.OFF,
  NA: constants.NA,
};

export const { DAILY_WITHDRAWLS } = constants;
export const { INTERNATIONAL_USAGE } = constants;
export const { DAILY_PURCHASE } = constants;
export const POSHVINE_ENCRYPTION_KEY = '8e990516e545ba4103dcb32a3462c54b';
export const POSHVINE_ENCRYPTION_IV = '39d0a599ea7c1761';

export const LIMIT = {
  PURCHASE: 'PURCHASE',
  ATM_WITHDRAWAL: 'ATM_WITHDRAWAL',
};

export const API_CONST = {
  REFERENCE_VALUE_C: 'C',
  REQUEST_TYPE: 'CardStatus',
  DATA_VERSION: 0,
};

export const STATUS = {
  PAY_BILL: [
    'CLSB',
    'CLSC',
    'CLSD',
    'CLSS',
    'DCSD',
    'DECL',
    'DELD',
    'UACC',
    'AOWO',
    'BLCK',
    'CHOF',
    'CRWO',
    'MOCB',
    'PCLS',
  ],
  ACTIVE_STATUS: ['NORM', 'NEW ', 'NEW'],
  TEMP_BLOCK_STATUS: ['FSFT'],
  TEMP_BLOCK: ['NORM', 'NEW ', 'NEW'],
  SET_PIN: ['DO00', 'DO99', 'NEW', 'NEW ', 'NORM'],
  REPLACE_CARD: ['DO00', 'DO99', 'NEW', 'NORM', 'LOCH', 'LOST', 'DOM1', 'DOM2', 'LOBK', 'FSFT', 'FHRD'],
  CLOSED_CARD: ['AOWO', 'CHOF', 'CLSB', 'CLSC', 'CLSD', 'CLSS', 'CRWO', 'DCSD', 'DELD', 'MOCB', 'UACC'],
};

export const ACTIVATION = {
  ACTIVE: 'A',
  INACTIVE: 'I',
};

export const BILL_DESK_URL = 'https://pgi.billdesk.com/pgidsk/pgmerc/idfccard/IDFCCARD.jsp';

export const PayementMode = {
  Nach: 'NACH Payments',
  nach: 'NACH Payments',
  Sipm: 'Standing Instruction',
  sipm: 'Standing Instruction',
  Imps: 'IMPS Payments',
  imps: 'IMPS Payments',
  neft: 'NEFT Payments',
  pupi: 'UPI Payments',
  bdsk: 'Payment Gateway',
  chqu: 'Clearing Cheque',
  pift: 'Internal Fund Transfer',
  cash: 'Cash Payments',
};
export const CC_ERROR_CODE = {
  ERR_CC_MULE: 'ERR_MULE_NULL_RESPONSE',
  ERR_MULE_SERVICE_DOWN: 'ERR_MULE_SERVICE_DOWN',
  ERR_MULE_POST_API: 'ERR_MULE_POST_API',
  ERR_MULE_RESPONSE_VALIDATION_FAILED: 'ERR_MULE_RESPONSE_VALIDATION_FAILED',
  MANDATORY_FIELD_MISSING: 'MANDATORY_FIELD_MISSING',
  ERR_USER_DETAILED_NOT_FOUND_IN_CONTEXT: 'ERR_USER_DETAILED_NOT_FOUND_IN_CONTEXT',
  ERR_CONVERTING_STRUCT_TO_MAP: 'ERR_CONVERTING_STRUCT_TO_MAP',
  ERR_INTERNAL_SERVER: 'ERR_INTERNAL_SERVER',
  ERR_DATA_TYPE_CONVERSION: 'ERR_DATA_TYPE_CONVERSION',
  ERR_CC_BAD_REQUEST: 'ERR_CC_BAD_REQUEST',
  ERR_JSON_UNMARSHAL: 'ERR_JSON_UNMARSHAL',
};
