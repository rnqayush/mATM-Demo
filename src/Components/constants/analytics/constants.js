export const PAGE_EVENT_STATUS = {
  ONGOING: 'ongoing',
  CLOSE: 'close',
  SUCCESS: 'success',
  ERROR: 'error',
  REJECT: 'reject',
};

export const PAGE_EVENT_TRACK = {
  CREATE_FD: {
    PAGE_NAME: 'createfd',
    LINK_NAME: {
      CLOSE: 'createfd_close',
      SUBMIT: 'createfd_submit',
    },
    PRODUCT_NAME: 'fixed deposit',
  },
  CREATE_RD: {
    PAGE_NAME: 'createrd',
    LINK_NAME: {
      CLOSE: 'createrd_close',
      SUBMIT: 'createrd_submit',
    },
    PRODUCT_NAME: 'recurring deposit',
  },
  ACCOUNT_LIST: {
    PAGE_NAME: 'accounts',
    LINK_NAME: {
      ADD_OR_CHANGE_NOMINEE: 'add_or_change_nominee',
      ACCOUNT_DETAILS: 'account_details',
      VIEW_TRANSACTIONS: 'view_transactions',
      ADD_FUNDS: 'add_funds',
      VIEW_STATEMENT: 'view_statement',
      REQUEST_CHEQUE_BOOK: 'request_cheque_book',
      MANAGE_CHEQUE_BOOK: 'manage_cheque_book',
      DOWNLOAD_STATEMENT: 'download_statement',
    },
  },
  ACCOUNT_DETAILS: {
    PAGE_NAME: 'AccountDetails',
    LINK_NAME: {
      ADD_OR_CHANGE_NOMINEE: 'add_or_change_nominee',
      VIEW_TRANSACTIONS: 'view_transactions',
      ADD_FUNDS: 'add_funds',
      VIEW_STATEMENT: 'view_statement',
      REQUEST_CHEQUE_BOOK: 'request_cheque_book',
      MANAGE_CHEQUE_BOOK: 'manage_cheque_book',
      CREATE_OD_AGAINST_FD: 'create_od_against_fd',
      MANAGE_OD_AGAINST_FD: 'manage_od_against_fd',
      DOWNLOAD_STATEMENT: 'download_statement',
    },
  },
  DEPOSITS: {
    PAGE_NAME: 'deposits',
    LINK_NAME: {
      DEPOSIT_DETAILS: 'deposit_details',
      CREATE_FD: 'create_fd',
      CREATE_RD: 'create_rd',
      CLOSE_FD: 'close_fd',
      CLOSE_RD: 'close_rd',
      ADD_OR_CHANGE_NOMINEE: 'add_or_change_nominee',
      DOWNLOAD_FD_ADVICE: 'download_fd_advice',
      DOWNLOAD_RD_ADVICE: 'download_rd_advice',
      EMAIL_FD_ADVICE: 'email_fd_advice',
      EMAIL_RD_ADVICE: 'email_rd_advice',
    },
  },
  DEPOSIT_DETAILS: {
    PAGE_NAME: 'DepositDetails',
    LINK_NAME: {
      CLOSE_FD: 'close_fd',
      CLOSE_RD: 'close_rd',
      ADD_OR_CHANGE_NOMINEE: 'add_or_change_nominee',
      DOWNLOAD_FD_ADVICE: 'download_fd_advice',
      DOWNLOAD_RD_ADVICE: 'download_rd_advice',
      EMAIL_FD_ADVICE: 'email_fd_advice',
      EMAIL_RD_ADVICE: 'email_rd_advice',
      CREATE_OD_AGAINST_FD: 'create_od_against_fd',
      MANAGE_OD_AGAINST_FD: 'manage_od_against_fd',
    },
  },
  DEBIT_CARD_LIST: {
    PAGE_NAME: 'cards',
    LINK_NAME: {
      DEBIT_CARD_DETAIL: 'debit_card_details',
      CHANGE_PIN: 'change_pin',
      GENERATE_PIN: 'generate_pin',
      TEMPORARY_BLOCK: 'temporary_block',
      PERMANENT_BLOCK: 'permanent_block',
      UNBLOCK: 'unblock',
      RE_ISSUE_CARD: 're_issue_card',
      REQUEST_VIRTUAL_CARD: 'request_virtual_card',
    },
  },
  DEBIT_CARD_DETAIL: {
    PAGE_NAME: 'DEBIT_CARD_DETAIL',
    LINK_NAME: {
      CHANGE_PIN: 'change_pin',
      GENERATE_PIN: 'generate_pin',
      TEMPORARY_BLOCK: 'temporary_block',
      PERMANENT_BLOCK: 'permanent_block',
      UNBLOCK: 'unblock',
      RE_ISSUE_CARD: 're_issue_card',
      CONVERT_TO_PHYSICAL_CARD: 'convert_to_physical_card',
      MANAGE_ATM_LIMIT: 'manage_atm_limit',
      MANAGE_PURCHASE_LIMIT: 'manage_purchase_limit',
      DEBITCARD_MANDATE: 'debitcard_mandate',
    },
  },
  INSURANCE_REDIRECTION_DISCLAIMER: {
    PAGE_NAME: code => `Wealth_Insurance_Redirection_Disclaimer_${code}_Proceed`,
    LINK_NAME: 'Proceed',
  },
  PAY_ABROAD: {
    LINK_NAME: {
      PAY_ABROAD_ACCOUNT_SELECT: 'pay_abroad_account_select',
    },
  },
};

export const CLICK_STREAM_TRACK = {
  OPEN_SAVINGS_ACCOUNT_LINK: 'open_savings_account_link',
  PROFILE_SWITCHING: 'profile_switching',
  REFER_AND_EARN_LINK: 'refer_and_earn_link',
  REFER_NOW_LINK: 'refer_now_explore_tab',
};
