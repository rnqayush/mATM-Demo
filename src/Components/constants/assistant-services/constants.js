export const VIDEO = 'Video';
export const CHAT = 'Chat';
