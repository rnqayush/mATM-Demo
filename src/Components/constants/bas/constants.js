export const USER_STATUS = {
  NOT_REGISTERED: 'NOT_REGISTERED',
  ACTIVE: 'ACTIVE',
  BLOCKED: 'BLOCKED',
  REMOVED: 'REMOVED',
};

export const CONTRACT_MATCHERS = {
  STRING: '[\\w\\s]{1,255}',
  NUMBER: '^\\d+$',
};

export const USER_ROLE_TYPE = {
  SOLE: 'SOLE',
};
