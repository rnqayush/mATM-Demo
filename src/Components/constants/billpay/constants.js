import { translate } from 'l10n';

export const FLOWS = {
  FETCH_BILL: 'FETCH_BILL',
  FETCH_BILLERS: 'FETCH_BILLERS',
  FETCH_OPERATOR: 'FETCH_OPERATOR',
  RECHARGE_PLANS: 'RECHARGE_PLANS',
};

export const FETCH_MY_BILLERS = 'Pay Bill - Saved Biller Get Billers';
export const FETCH_PENDING_BILLS = 'Pay Bill - Saved Biller Get Pending Bills';
export const CC_BOC = '1860 500 1111';

export const AUTOPAY_TYPE = {
  ENABLE: 'Enable',
  MODIFY: 'Modify',
};

export const AUTOPAY_SUPPORT_TEXT = {
  Enable: [
    translate('PAY.BILL_PAY.AUTOPAY_SUPPORT_MESSAGE.ENABLE_AUTOPAY'),
    translate('PAY.BILL_PAY.AUTOPAY_SUPPORT_MESSAGE.MANUALLY_PAY_BILLS_EXCEEDING_AMOUNT'),
  ],
  Modify: [
    translate('PAY.BILL_PAY.AUTOPAY_SUPPORT_MESSAGE.MODIFY_AUTOPAY'),
    translate('PAY.BILL_PAY.AUTOPAY_SUPPORT_MESSAGE.MANUALLY_PAY_BILLS_EXCEEDING_AMOUNT'),
  ],
};

export const billpayTransactionURL = uuid => `/billpay/${uuid}`;
