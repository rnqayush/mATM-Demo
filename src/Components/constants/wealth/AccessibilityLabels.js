const accessibilityLabels = {
  SGB_HOLDINGS_SHARE: 'Double tap to share certificates',
};

export default accessibilityLabels;
