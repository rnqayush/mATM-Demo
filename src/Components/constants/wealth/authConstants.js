const PERMISSIONS = {
  WEALTH_BUY_JOURNEY: 'WEALTH_BUY_JOURNEY',
  BUY_MUTUAL_FUND_V2: 'BUY_MUTUAL_FUND_V2',
  MANAGE_MUTUAL_FUNDS: 'MANAGE_MUTUAL_FUNDS',
  INVESTMENT: 'INVESTMENTS',
  SGB: 'SGB',
  RISK_PROFILE: 'RISK_PROFILE',
  INSURANCE: 'INSURANCE',
  OPEN_DEMAT: 'OPEN_DEMAT',
  OPEN_IPO: 'OPEN_IPO',
  INVEST_IPO: 'INVEST_IPO',
  VIEW_TRANSACTIONS: 'VIEW_TRANSACTIONS',
  BUY_MUTUAL_FUNDS: 'BUY_MUTUAL_FUNDS',
  BUY_SGB: 'BUY_SGB',
  OPEN_DEMAT_VENDOR: 'OPEN_DEMAT_VENDOR',
  BUY_SGB_ACTION: 'BUY_SGB_ACTION',
  PIS_DOC: 'PIS_DOC',
};

export default PERMISSIONS;
