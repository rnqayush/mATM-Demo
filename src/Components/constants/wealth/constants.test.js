import { TRACING } from 'constants/wealth/constants';

describe('constants', () => {
  it('should match TRACING snapshot', () => {
    expect(TRACING).toMatchSnapshot();
  });
});
