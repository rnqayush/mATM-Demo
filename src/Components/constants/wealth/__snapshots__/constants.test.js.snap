// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants should match TRACING snapshot 1`] = `
Object {
  "ALTERNATE_HOLDINGS": "Get Alternates Holdings",
  "BUY_MUTUAL_FUND_CART": "Buy Mutual Fund Cart",
  "BUY_RECOMMENDED_MUTUAL_FUND": "Buy Recommended Mutual Fund",
  "BUY_SINGLE_LUMPSUM": "Buy single mutual fund with lumpsum",
  "BUY_SINGLE_LUMPSUM_PLUS_SIP": "Buy single mutual fund with sip and lumpsum",
  "BUY_SINGLE_SIP": "Buy single mutual fund with sip",
  "DEMAT_CRM_DETAILS": "Demat CRM Details",
  "DEMAT_DIS_DETAILS": "Demat DIS Details",
  "DEMAT_TERMS_AND_CONDITIONS": "Get terms and condition for demat journey",
  "DEMAT_TERMS_AND_CONDITIONS_1": "Get terms and condition for Demat Schedule of Charges",
  "DEMAT_TERMS_AND_CONDITIONS_2": "Get terms and condition for Demat Rights and Obligations of Beneficial Owner and Depository Participant as prescribed by SEBI and Depositories",
  "DOWNLOAD_SGB_CERT": "Download SGB Certificate",
  "EDIT_SIP": "Edit Existing SIP",
  "FETCH_BANK_INV_ACCOUNT": "Get Bank & Investment accounts",
  "FULL_FUND_REDEEM": "MF Full Redeem",
  "FULL_FUND_SWITCH": "Full switch MF",
  "GET_BANK_ACCOUNTS": "Get Bank accounts",
  "GET_COMPUTED_DATA_FOR_GOAL_CATEGORY_DETAILS": "Get computed data for Goal Category details",
  "GET_COMPUTED_FUTURE_VALUE": "Get computed future value for goal recommendatio",
  "GET_CURRENT_AVAILABLE_IPO_LIST": "Get Current Available IPO List",
  "GET_CUSTOMER_CURRENT_ASSETS": "Get Customer Holding Current Assets",
  "GET_CUSTOMER_HOLDINGS": "Get Mutual Fund Holdings",
  "GET_CUSTOMER_SIGNATURE": "Get customer signature",
  "GET_DEFAULT_POLICY_TERM": "Get default policy term based on customer age",
  "GET_DEMAT": "Get Demat Details",
  "GET_DEMAT_ACCOUNT_DEBITS": "Get demat account debits",
  "GET_DEMAT_ACCOUNT_DETAILS": "Get Demat Account Details",
  "GET_DEMAT_ENTRY_POINT_DETAILS": "Get Demat Entry Point Details",
  "GET_DEMAT_NSDL_REDIRECTION_DETAILS": "Get Demat NSDL Redirection Details",
  "GET_DEMAT_SINGUP_LINK": "Get Demat Signup Link",
  "GET_FATCA_COUNTRIES": "Get FATCA Countries",
  "GET_FATCA_MASTER": "Get FATCA Master",
  "GET_FATCA_TNC": "Get FATCA TNC",
  "GET_FUND_HOLDING_PATTER": "Get Fund Holding pattern",
  "GET_FUND_PERFORMANCE": "Get Fund Performance",
  "GET_GOAL_CATEGORY": "Get Goal Categories",
  "GET_GOAL_CATEGORY_DETAILS": "Get Goal Category details",
  "GET_GOAL_INVESTMENT_DETAILS": "Get investment details of a goal",
  "GET_HEALTH_INSURANCE_COVER_AMOUNT": "Get Health Insurance Cover Amount",
  "GET_HEALTH_INSURANCE_STATIC_DATA": "Get Health Insurance Static Data",
  "GET_INSURANCE_CATEGORIES": "Get insurance categories",
  "GET_INSURANCE_CATEGORIES_AEM": "Get insurance categories AEM data",
  "GET_INSURANCE_REDIRECTION_URL": "Get insurance redirection url",
  "GET_INSURANCE_SUB_CATEGORIES_AEM": "Get insurance sub categories AEM data",
  "GET_INSURANCE_SUMMARY": "Get summary for insurance dashboard",
  "GET_INVESTMENT_ACCOUNTS": "Get Investment accounts",
  "GET_ISEC_ACCESS_DETAILS": "Get Isec Access Details",
  "GET_ISEC_DETAILS": "Get ICICI Securities details",
  "GET_ISEC_OTP_DETAILS": "Get ICICI Securities send otp details",
  "GET_ISEC_RESEND_OTP_DETAILS": "Get ICICI Securities resend otp details",
  "GET_KYC_DETAILS": "Get KYC details",
  "GET_MF_PORTFOLIOGRAPH": "Get Mutual Fund Portfoliograph",
  "GET_MF_REPORT_FIN_YEARS": "Get Mutual fund report financial years",
  "GET_MF_SELLABLE_QUANTITY": "Get mutual fund sellable quantity",
  "GET_MF_SUB_ASSET_CLASSES_STATIC_DATA": "Get Mutual Fund Sub Asset Classes Static Data",
  "GET_MF_TRANSACTIONS": "Get Mutual Fund Transactions",
  "GET_MONTHLY_SAVINGS": "Get monthly savings",
  "GET_MUTUAL_FUNDS": "Get Mutual Funds",
  "GET_MUTUAL_FUND_AMC": "Get Mutual Fund AMCs",
  "GET_MUTUAL_FUND_ASSETS": "Get Mutual Fund Asset class filters",
  "GET_MUTUAL_FUND_MAX_AMOUNT_CONFIG": "Get Mutual Fund Maximum Amount configuration",
  "GET_NEW_FUND_OFFERS": "Get new fund offers",
  "GET_PIS_ACCOUNTS_DEATILS": "Get PIS Accounts Details",
  "GET_RBI_SECURITIES": "Get RBI Securities Content",
  "GET_RECOMMENDATION_CONFIG": "Get Recommendation Config",
  "GET_RISK_PROFILE": "Get Risk Profile",
  "GET_RISK_PROFILE_CATEGORIES": "Get Risk Profile Categories",
  "GET_RISK_PROFILE_DETAILS_FOR_GOALS": "Get Customer Risk Profile and Categories",
  "GET_RISK_PROFILE_QUESTIONS": "Get Risk Profile Questions",
  "GET_SCHEME_DETAILS": "Get Scheme Details",
  "GET_SCHEME_RISK_PROFILE": "Get Scheme Riks Profile",
  "GET_SELECTED_AVAILABLE_IPO_LIST": "Get Selected Available IPO List",
  "GET_SGB_BUY_DETAILS": "Get account,demat Details to buy SGB",
  "GET_SGB_TRANSACTION": "Get SGB Transactions",
  "GET_SYS_PLANS": "Get systematic plan",
  "GET_TERM_CONFIG": "Get Term Insurance Config",
  "GET_TERM_PLAN_QUOTE": "Get quote for term plan from each vendor",
  "GET_TRANSACTION_DETAIL": "Get MF Transaction details",
  "GET_USER_DETAILS": "Get User Details",
  "GET_ZERODHA_ACCESS_DETAILS": "Get zerodha access details",
  "GET_ZERODHA_ACCOUNT_LIST": "Get Zerodha Account List",
  "GET_ZERODHA_ACTIVE": "Get Zerodha Active Status",
  "GET_ZRODHA_OTP_DETAILS": "Get zerodha send otp details",
  "GET_ZRODHA_RESEND_OTP_DETAILS": "Get zerodha resend otp details",
  "GOAL_BUY": "Buy Goals",
  "GOAL_RECOMMENDED_FUND": "Goal Recommended funds",
  "GOAL_TRACKING": "Get goal tracking list",
  "HOLDING_ASSET_ALLOCATION": "Get Asset allocation",
  "HOLDING_PRODUCT_ALLOCATION": "Get Product allocation",
  "HOLDING_SUMMARY": "Get Holding Summary",
  "INVEST_INTO_IPO": "IPO Place order",
  "IPO_APPLICANTS_INFO": "IPO Applicants info",
  "IPO_CREATE_APPLICANTS": "IPO Create Applicant",
  "IPO_DOWNLOAD_PDF": "IPO Download Pdf",
  "IPO_MODIFY_IPO": "IPO Modify IPO",
  "IPO_SEND_EMAIL": "IPO Send Email",
  "LOAD_WEALTH_DASHBOARD": "Load Wealth Dashboard",
  "MF_SWITCH_REDEEM_SIGNATURE_CHECK": "MF Switch/Redeem signature check",
  "PARTIAL_FUND_REDEEM": "MF Partial Redeem",
  "PARTIAL_FUND_SWITCH": "Partial switch MF",
  "PAUSE_SIP": "Pause Existing SIP",
  "PIS_CAPITAL_GAIN_SEND_EMAIL": "PIS capital gain send email",
  "PIS_HOLDING_STATEMENT_SEND_EMAIL": "PIS holding statement send email",
  "PIS_PERMISSION_LETTER_SEND_EMAIL": "PIS permission letter send email",
  "PLACE_SGB_ORDER": "Buy Sovereign Gold Bond",
  "POST_FATCA_TNC": "Post FATCA TNC",
  "RESUME_SIP": "Resume SIP",
  "SGB_ACTIVE_TRANCH": "Get SGB Active Tranch",
  "SHARE_SGB_CERT": "Share SGB Certificate",
  "STOP_STP": "Stop STP",
  "STOP_SWP": "Stop SWP",
  "SUBMIT_FATCA": "Submit FATCA",
  "SUBMIT_PAN_NUMBER": "Submit Pan Number",
  "SUBMIT_RISK_PROFILE": "Submit Risk Profile",
  "SYS_FUND_REDEEM": "MF Systematic Redeem",
  "SYS_FUND_SWITCH": "Start STP",
  "TERMS_AND_CONDITIONS": "Get terms and condition for wealth journy",
  "UPLOAD_PAN": "Upload PAN Card",
  "WITHDRAW_IPO": "Withdraw IPO",
}
`;
